src/security/index.ts

```diff
@@ -22,6 +23,7 @@ export default class Security extends Test {
       new Cookies(),
       new ReferrerPolicy(),
       new PermissionsPolicy(),
+      new ContentEncoding()
```

Comment:
Please, add comma (`,`) in the end of the line.

https://github.com/airbnb/javascript#commas